global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "5.0.404"
+  }
```

Comment:
Add `"rollForward": "patch"` so this doesn't break for minor updates on build machines?